.env.example

```
@@ -1,3 +1,25 @@
+LEADS_EMAIL=jsb@synsol.ch
+
+# Consent gating
+CONSENT_MODE=optional   # required|optional|off
+CONSENT_LINE="This call may be recorded and summarized to create a trading lead. Continue?"
+
+# Retention
+RETENTION_DAYS=-1
+
+# Google Sheets
+GOOGLE_SHEETS_ID=
+GOOGLE_SERVICE_ACCOUNT_JSON={}
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GOOGLE_SERVICE_ACCOUNT_JSON key should go before the GOOGLE_SHEETS_ID key

(UnorderedKey)